server/pbench/bin/index-pbench

```diff
@@ -618,6 +631,133 @@ _known_csv_handlers = {
             'metadata_pat': re.compile(r'(?P<pid>.+?)-(?P<command>.+)')
         }
     },
+    'proc-vmstat': {
+        '@prospectus': {
+            # For proc-vmstat .csv files, we want to individually index each
+            # column entry as its own JSON document.
+            'handling': 'individual'
+        },
+        'proc-vmstat_allocstall_delta_sec.csv': {
```

Comment:
Yes I think it would be easier to parse the `proc-vmstat-stdout.txt` file for this information rather than the CSV files. I have a prototype that does this and creates a more flat data structure.